config/custom-environment-variables.json

```diff
@@ -10,5 +10,10 @@
     "port": "COUCHDB_PORT",
     "username": "COUCHDB_USER",
     "password": "COUCHDB_PASS"
+  },
+  "gitlabOAuth": {
+    "clientID": "GITLAB_CLIENT_ID",
+    "clientSecret": "GITLAB_CLIENT_SECRET",
+    "callbackURL": "GITLAB_CALLBACK_URL"
```

Comment:
Am I right in assuming that the callback URL can be anything, like `http://localhost:5000/secretPage`?

Reply:
yes, in this case [`http://127.0.0.1:5000/auth`](https://github.com/jum-s/data.transformap.co/blob/425-oauth2-provider/server/controllers/routes.js#L23)